runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -696,7 +813,11 @@ private void rebuildSkills()
 		// Rebuild calculated values like xp/hr in panel
 		for (Skill skill : Skill.values())
 		{
-			xpPanel.updateSkillExperience(false, xpPauseState.isPaused(skill), skill, xpState.getSkillSnapshot(skill));
+			final boolean updated = false;
+			final boolean paused = xpPauseState.isPaused(skill);
+			final XpSnapshotSingle xpSnapshotSingle = xpState.getSkillSnapshot(skill);
+
+			xpPanel.updateSkillExperience(updated, paused, skill, xpSnapshotSingle);
```

Comment:
What's going on here?

Reply:
There's too much going on in one line which made it difficult to read. Variables aren't required but I would at least recommend line breaks. 

```java
xpPanel.updateSkillExperience(
    false,
    xpPauseState.isPaused(skill),
    skill,
    xpState.getSkillSnapshot(skill));
```

Personal preference: I dislike having to rely on mouseover in an IDE to read code. Specifically the loose `false` is ambiguous without more context.